lib/atlassian-jwt-authentication/verify.rb

```diff
@@ -57,7 +57,7 @@ def verify
         end
 
         decode_key = OpenSSL::PKey::RSA.new(response.body)
-        decode_options = {algorithms: ['RS256']}
+        decode_options = {algorithms: ['RS256'], verify_aud: true, aud: "https://" + Rails.application.secrets.http_host}
```

Comment:
That's not a proper solution because it ties AJA with Rails and specific file (secrets.yml).

I think this should be solve by either:
- adding an additional argument to verify_jwt method
- adding new configuration option in AtlassianJwtAuthentication

Either way would be a better approach.